spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+RSpec.describe 'User Show', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'Akin',
+      bio: 'photographer',
+      photo: 'http://passport.com',
+      post_counter: 0
+    )
+    @post1 = Post.create(
+      author: @user,
+      title: 'My test post',
+      text: 'This is my test post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+    @post2 = Post.create(
+      author: @user,
+      title: 'My second test post',
+      text: 'This is my second post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+  describe 'a user show page' do
+    it 'displays the users profile photo' do
+      visit user_path(@user)
+      expect(page).to have_css("img[src*='http://passport.com']")
+    end
+  end
+  it 'displays the user name' do
+    visit user_path(@user)
+    expect(page).to have_content('Akin')
+  end
+  it 'displays the user bio' do
+    visit user_path(@user)
+    expect(page).to have_content('photographer')
+  end
+  it 'shows the right Number of posts' do
+    visit user_path(@user)
+    expect(page).to have_content('Number of posts: 2')
+  end
```

Comment:
- [ ] It is a requirement to have a **test that asserts you can see the user's first 3 posts.**. Kindly add more than 3 posts and make sure to test that you can see only the users first 3 posts

Reply:
- [ ] Kindly add a _tests that asserts you can see a **button** that lets me view all of a user's posts._ Meaning that you should have a button on your test that when clicked you can see all the users post. Add a button